src/lib/logical_query_plan/export_node.hpp

```diff
@@ -14,11 +14,10 @@ namespace hyrise {
  */
 class ExportNode : public EnableMakeForLQPNode<ExportNode>, public AbstractNonQueryNode {
  public:
-  ExportNode(const std::string& init_table_name, const std::string& init_file_name, const FileType init_file_type);
+  ExportNode(const std::string& init_file_name, const FileType init_file_type);
 
   std::string description(const DescriptionMode mode = DescriptionMode::Short) const override;
 
-  const std::string table_name;
   const std::string file_name;
   const FileType file_type;
```

Comment:
Not in scope of this PR but couldn't they be protected?

Reply:
We often do not protect members of LQP nodes when they do not change and are accessed for creating the corresponding operator. IMO I think it's okay to have these const public members rather than private/protected member + public accessor for these intermediate representations